product_multi_image/README.rst

```diff
@@ -0,0 +1,70 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===========================
+Multiple Images in Products
+===========================
+
+This module implements the possibility to have multiple images for a product,
+that it's to say, an image gallery.
+
+Usage
```

Comment:
Add Installation section talking about the dependency to the base module and in which repo it is.